docs/UserGuide.md

```diff
@@ -276,7 +276,21 @@ will only show an error message (`No new changes has been made to the data`).
 Likewise, entering `undo` with no prior changes will also display the same error message.
 
 **Note:** `undo` does not affect commands that do not modify the contact list
-(`help`, `list`, `find` etc.). 
+(`help`, `list`, `find` etc.).
+
+### Examples of Incorrect Inputs
+
+1. **Example 1: Empty Tag in Add Command**
+
+   If you enter the command `add n/John Doe p/12345678 e/johndoe@example.com r/Student a/PGPR c/CS2103T t/`, 
+   you will see an error message because the `t/` tag is empty. 
```

Comment:
It would be better to show which error message.

Reply:
I think can do both. But writing down the error message is sth necessary, and providing a screenshot is optional. But you can do what you think right, just make sure to show a clear result when user typed wrongly.